### PR TITLE
Make modrinth zip filename match CF zip

### PR DESCRIPTION
**Pull request by RozeFound**

### CODE REVIEW

**Comment by RozeFound** [comment]
I hope this won't break anything

File: CLI tools/CurseForge to Packwiz-Modrinth.py [hunk 1]
```diff
@@ -87,7 +87,10 @@ def extract_file(from_zip, from_file, to_path, from_desc, to_desc):
     mmc_zip_root = str(Path(cf_zip_path).parents[0])
     mmc_zip_path = mmc_zip_root + "\\Fabulously Optimized " + pack_version + ".zip"
     modrinth_config = git_path + "Modrinth\\mmc-export.toml"
-    os.system("mmc-export -i \"" + mmc_zip_path + "\" -f Modrinth --modrinth-search loose -o \"" + mmc_zip_root + "\" -c \"" + modrinth_config + "\" -v " + pack_version + " --scheme \"{abbr}_{name}_{version}\"")
+
```
Comment (Madis0) [inline CLI tools/CurseForge to Packwiz-Modrinth.py:90]: I can see the `\` is missing after `mmc_zip_path`. May have more such things, haven't tested

Comment (Madis0) [reply CLI tools/CurseForge to Packwiz-Modrinth.py:90]: oops meant to comment above the line below lol

Comment (RozeFound) [reply CLI tools/CurseForge to Packwiz-Modrinth.py:90]: wdym

Comment (Madis0) [reply CLI tools/CurseForge to Packwiz-Modrinth.py:90]: `mmc_zip_path + "\" -f Modrinth` to `mmc_zip_path\ -f Modrinth` I believe

Comment (RozeFound) [reply CLI tools/CurseForge to Packwiz-Modrinth.py:90]: More likely quote escape ¯\\_(ツ)\_\/¯

**Pull request merged**